internal/ethapi/transaction_args.go

```diff
@@ -72,6 +72,9 @@ type TransactionArgs struct {
 	Commitments []kzg4844.Commitment `json:"commitments"`
 	Proofs      []kzg4844.Proof      `json:"proofs"`
 
+	// For SetCodeTxType
+	AuthList []types.Authorization `json:"authList"`
```

Comment:
it is named `authorizationList` in other types (e.g. `toJSON`, `RPCTransaction`)
I think it is better to use use the same name in all json representations

Reply:
Thanks, I updated it in [618fcea](https://github.com/ethereum/go-ethereum/pull/30078/commits/618fcea505eefbc5dc1221bb4d40bd54b603ebaf)